packages/core/core/src/services/document-service/transform/sort.ts

```diff
@@ -0,0 +1,6 @@
+export const transformSort = (sort: string[] | string): string[] | string => {
+  if (Array.isArray(sort)) {
+    return sort.map((item) => (item === 'id' ? 'documentId' : item));
+  }
+  return sort.replace('id', 'documentId');
+};
```

Comment:
could you create a jira ticket to handle nested sorts 🙏 ? That way we don't lose track of that

Reply:
Do you mean a format like  { sort: "category.id" } should become  { sort: "category.documentId" } ?